scripts/cli/runInDocker.sh

```diff
@@ -76,7 +76,7 @@ build_temp_runner_image() {
   local SET_DOCKER_GID=$(getent group docker | cut -d: -f3)
   TEMP_RUNNER_IMAGE="sleeper-runner:$TEMP_TAG"
   echo "Propagating current user to Docker image"
-  docker build "$RUNNER_PATH" --quiet -t "$TEMP_RUNNER_IMAGE" \
+  docker build "$RUNNER_PATH" -t "$TEMP_RUNNER_IMAGE" \
```

Comment:
For me removing the quiet flag seems to result in too much output for the simplest CLI operation. I'd rather just close this and leave the bug open until we find another way. You could take it off the milestone for now, but please add any information you've found about the problem.

Do you think this change is a good idea?

Reply:
Fix is useful if the details provided have a meaningful impact on the issue.
Given that the information returned from the interaction without the flag is large, my view would align with yours that the output is excessive for the cli operation. Will close down review as not needed and provide details on parent tickets around conclusion so possible discussion of what might be needed for another approach